horizon/config.py

```diff
@@ -103,6 +104,24 @@ def __new__(cls, prefix=None, is_model=True):
     # enable datadog APM tracing
     ENABLE_MONITORING = confi.bool("ENABLE_MONITORING", False)
 
+    ENABLE_OFFLINE_MODE = confi.bool(
+        "ENABLE_OFFLINE_MODE",
+        False,
+        description="if true, sidecar will use a file backup to restore configuration and policy data when cloud services are unavailable",
+    )
+
+    OFFLINE_MODE_BACKUP_PATH = confi.str(
+        "OFFLINE_MODE_BACKUP_PATH",
+        "/app/backup/sidecar_config.json",
+        description="Path to backup sidecar cloud configuration to when in offline mode",
+    )
+
+    CONFIG_FETCH_MAX_RETRIES = confi.int(
+        "CONFIG_FETCH_MAX_RETRIES",
+        6,
```

Comment:
The current hard-coded value is 10, any real reason to change this ?

Reply:
If someone wants to fallback to backed up configuration when cloud isn't available (aka "offline mode") - then 10 retries with a max wait of 10 secs felt like a lot to me